src/main/java/lotto/constants/Winning.java

```diff
@@ -0,0 +1,49 @@
+package lotto.constants;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum Winning {
+    FIRST(6, 2000000000),
+    SECOND(5, 1500000),
+    THIRD(4, 50000),
+    FOURTH(3, 5000),
```

Comment:
```suggestion
    FIRST(6, 2_000_000_000),
    SECOND(5, 1_500_000),
    THIRD(4, 50_000),
    FOURTH(3, 5_000),
```

숫자가 너무 크다 싶으면 3글자마다 밑줄로 구분해보는건 어떨까요?
가독성이 좋아집니다.